.github/workflows/iroha2-custom-image.yml

```diff
@@ -41,15 +42,19 @@ env:
 
 jobs:
   build_executor:
-    runs-on: ubuntu-latest
+    runs-on: [self-hosted, Linux, iroha2]
     container:
       image: hyperledger/iroha2-ci:nightly-2024-09-09
     timeout-minutes: 30
     steps:
       - uses: actions/checkout@v4
         with:
           ref: ${{ github.event.inputs.CHECKOUT_REF }}
-      - uses: Swatinem/rust-cache@v2
```

Comment:
May be to continue to use the old rust cache Action if the new one doesn't always work?

Reply:
The old one is broken. It never hits the cache despite writing to it